### PR TITLE
Update in line with OpenFisca-UK

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Following some substantial refurbishment of OpenFisca-UK, this repo has been updated to ensure compatibility.

### CODE REVIEW

File: frs_params.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    "carers_allowance",
+    "BSP",
+    "JSA_contrib",
+    "ESA_contrib",
+    "state_pension",
+    "PIP_DL",
+    "PIP_M"
+]
+
+period_codes = {
```
Comment (MaxGhenis) [inline frs_params.py]: should this be caps too as a constant?

File: frs_params.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    7: "EAST_OF_ENGLAND",
+    8: "LONDON",
+    9: "SOUTH_EAST",
+    10: "SOUTH_WEST",
+    11: "WALES",
+    12: "SCOTLAND",
+    13: "NORTHERN_IRELAND"
+}
+
+AVERAGE_COUNCIL_TAX = [
```
Comment (MaxGhenis) [inline frs_params.py]: Could this be a dict or dataframe to clarify the connection to each region?

File: generate_csv.py [hunk 3]
```diff
@@ -2,126 +2,46 @@
...
+    "FRS_net_income"
+] + [benefit for benefit in BENEFITS.values() if benefit not in BENUNIT_LEVEL_BENEFITS and benefit in REPORTED] + [benefit + "_reported" for benefit in BENEFITS.values() if benefit not in BENUNIT_LEVEL_BENEFITS and benefit in SIMULATED and benefit]
 
 BENUNIT_FIELDNAMES = [
     "benunit_id",
-    "benunit_weight",
-    "external_child_maintenance"
-]
+    "benunit_weight"
+] + [benefit for benefit in BENEFITS.values() if benefit in BENUNIT_LEVEL_BENEFITS and benefit in REPORTED] + [benefit + "_reported" for benefit in BENEFITS.values() if benefit in BENUNIT_LEVEL_BENEFITS and benefit in SIMULATED]
```
Comment (MaxGhenis) [inline generate_csv.py]: any way to shorten this line?

**Review by MaxGhenis — COMMENTED** [review]
minor stuff

File: generate_csv.py (hunk 3 above)
Comment (nikhilwoodruff) [reply generate_csv.py]: Just used the formatter on it, should add that as a GitHub action for this repo

**Pull request merged**